.github/workflows/push-pull-main.yml

```diff
@@ -93,21 +63,25 @@ jobs:
 
 
   unit-tests:
-    needs: commit-updated-env
     runs-on: ${{ matrix.operating-system }}
     strategy:
       matrix:
         include:
           - operating-system: macos-latest
-            python-version: '3.10'
-            label: osx-64-py-3-10
+            python-version: '3.11'
+            label: osx-64-py-3-11
```

Comment:
Version updates are fine.

I'd love to get the python version centralized somewhere so we only need to update it in one spot, but I never figured out how to get that sort of centralization working. For now the manual bump to 11 is 👍